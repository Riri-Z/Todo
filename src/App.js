import React from 'react';
import styled from 'styled-components'
import NavbarComponent from './components/Navbar';
import Container from '@material-ui/core/Container';
import TaskContextProvider from './contexts/TaskContext';
import TodoListComponent from './components/TodoList';

const Wrapper = styled.section`
  padding: 4em;
  background: #c1e5ff;
`;

function App() {
  return (
    <Wrapper>
      <Container fixed>
        <TaskContextProvider>
          <NavbarComponent />
          <TodoListComponent />
        </TaskContextProvider>
      </Container>
    </Wrapper>

  );
}

export default App;