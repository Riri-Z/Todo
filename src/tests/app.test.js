import React from 'react'
import renderer from 'react-test-renderer'
import App from '../App'
test('App List renders correctly', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})