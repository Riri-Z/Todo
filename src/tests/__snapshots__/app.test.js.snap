// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App List renders correctly 1`] = `
<section
  className="sc-AxjAm bFBxWV"
>
  <div
    className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-dense MuiToolbar-gutters"
        style={
          Object {
            "justifyContent": "center",
          }
        }
      >
        <h6
          className="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
        >
          React Todo List (
          0
          )
        </h6>
      </div>
    </header>
    <div
      className="makeStyles-root-1"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <form
            onSubmit={[Function]}
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
            >
              <div
                className="MuiFormControl-root MuiTextField-root"
              >
                <label
                  className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-marginDense MuiInputLabel-outlined"
                  data-shrink={false}
                >
                  Add Task
                </label>
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-marginDense MuiOutlinedInput-marginDense"
                  onClick={[Function]}
                >
                  <input
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMarginDense MuiOutlinedInput-inputMarginDense"
                    disabled={false}
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    required={false}
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-2 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      className="PrivateNotchedOutline-legendLabelled-4"
                    >
                      <span>
                        Add Task
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
          </form>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <ul
            className="MuiList-root makeStyles-root-6 MuiList-padding"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
